rero_ils/modules/documents/dumpers/indexer.py

```diff
@@ -240,6 +240,8 @@ def _process_files(self, record, data):
         files = {}
         for record_file in record.get_records_files():
             collections = record_file.get('metadata', {}).get('collections')
+            owners = record_file.get('metadata', {}).get('owners')
+            library_pid = owners[0].replace('lib_', '')
```

Comment:
this raise an exception if `owners` is `None`

Reply:
You are right. However for the moment owners should be there. I probably create a new PR at the end of the US and refactor the `links` and `owners` properties.